src/server.cpp

```diff
@@ -1076,8 +1076,7 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
                     // stereo: apply stereo-to-mono attenuation
                     for ( i = 0, k = 0; i < iServerFrameSizeSamples; i++, k += 2 )
                     {
-                        vecdIntermProcBuf[i] +=
-                            ( static_cast<double> ( vecsData[k] ) + vecsData[k + 1] ) / 2;
+		        vecdIntermProcBuf[i] += ( vecsData[k] + vecsData[k + 1] ) / 2.0f;
```

Comment:
Why do you need 2.0f here? the vecfData is already float.

Reply:
Fixed.